Dockerfile

```diff
@@ -43,8 +43,7 @@ RUN apk add --no-cache bash openssh ca-certificates jq curl openssl perl git zip
 ARG JP_VERSION="0.1.3"
 
 RUN curl -L https://github.com/jmespath/jp/releases/download/${JP_VERSION}/jp-linux-amd64 -o /usr/local/bin/jp \
- && chmod +x /usr/local/bin/jp \
- && pip install --no-cache-dir --upgrade jmespath-terminal
+ && chmod +x /usr/local/bin/jp
```

Comment:
https://github.com/Azure/azure-cli/pull/21206 is a breaking change and should be included in release notes.

Reply:
Updated